provider/ec2/local_test.go

```diff
@@ -409,8 +409,8 @@ func (t *localServerSuite) TestSystemdBootstrapInstanceUserDataAndState(c *gc.C)
 	// check that a new instance will be started with a machine agent
 	inst1, hc := testing.AssertStartInstance(c, env, t.ControllerUUID, "1")
 	c.Check(*hc.Arch, gc.Equals, "amd64")
-	c.Check(*hc.Mem, gc.Equals, uint64(3.75*1024))
-	c.Check(*hc.CpuCores, gc.Equals, uint64(1))
+	c.Check(*hc.Mem, gc.Equals, uint64(1024))
```

Comment:
this sounds like our default instance type is going away from a 3.75GB machine with 1 CPU to a 1GB machine with 2 CPUs, which probably isn't the best tradeoff, is it?

Reply:
Yeah, that's why I'm uncomfortable to land this. This seems like a big tradeoff